api/routes/device.go

```diff
@@ -322,9 +322,10 @@ func (h *Handler) UpdateDevice(c gateway.Context) error {
 		tenant = c.Tenant().ID
 	}
 
-	if err := guard.EvaluatePermission(c.Role(), guard.Actions.Device.Update, func() error {
+	err := guard.EvaluatePermission(c.Role(), guard.Actions.Device.Update, func() error {
 		return h.service.UpdateDevice(c.Ctx(), tenant, models.UID(req.UID), req.Name, req.PublicURL)
-	}); err != nil {
+	})
+	if err != nil {
```

Comment:
Why do you have preferred changed how the if was structured?

Reply:
I strongly believe that the syntax of `if err := ...; err != nil {}` becomes highly illegible when the `[..]; err != nil {}` statement is placed on another line or when the function call is too long. In my opinion, the eyes get lost when trying to understand the flow of the code.

Reply:
This has been my experience, and it is why I consider it a good idea to change this.

Reply:
I believe binding it to the `if` statement also reduces the variable's scope. I prefer the original code (even if it's uglier).

Reply:
> I believe binding it to the `if` statement also reduces the variable's scope. I prefer the original code (even if it's uglier).

I agree. This change could fit perfectly if we needed the error in the following, but we don't.

Reply:
I understand your concerns; I'm rolling back this commit.